object/test/generate.go

```diff
@@ -571,3 +572,16 @@ func GenerateGetRangeHashResponse(empty bool) *object.GetRangeHashResponse {
 
 	return m
 }
+
+func GenerateLock(empty bool) *object.Lock {
```

Comment:
What about providing object IDs as arguments instead of `empty`?

Reply:
I don't think it makes sense because then function becomes trivial. The meaning of all these generators is the lack of concern about how exactly the fields are filled.